m2cgen/interpreters/interpreter.py

```diff
@@ -89,9 +89,14 @@ class ToCodeInterpreter(BaseToCodeInterpreter):
     about AST.
     """
 
+    exponent_function_name = NotImplemented
+    power_function_name = NotImplemented
+    tanh_function_name = NotImplemented
```

Comment:
`NotImplemented` is an alias to the `None`, and while in the code it looks good, in runtime we would get a native code which attempts to call function called `None`.

The best thing I can think of is just adding an assert to verify that the corresponding function is not empty.

Reply:
Will do. I just followed the same logic as in https://github.com/BayesWitnesses/m2cgen/blob/master/m2cgen/interpreters/code_generator.py#L26-L34

Reply:
Fixed